### PR TITLE
bugfix: add fallback array to foreach

### PR DESCRIPTION
**Pull request by Karol-Stelmaczonek**
Ticket: https://oat-sa.atlassian.net/browse/INF-212

## What's Changed

- Add fallback array for foreach

### CODE REVIEW

**Review by andreluizmachado — APPROVED** [review]
LGTM

File: helpers/CacheHelper.php [hunk 1]
```diff
@@ -137,7 +137,7 @@ public static function getControllerAccess($controllerClassName)
             }
 
             // roles by action
-            foreach (ControllerHelper::getActions($controllerClassName) as $actionName) {
+            foreach (ControllerHelper::getActions($controllerClassName) ?? [] as $actionName) {
```
Comment (pribi) [inline helpers/CacheHelper.php:140]: Can ControllerHelper::getActions($controllerClassName) return other than null or array?

Comment (Karol-Stelmaczonek) [reply helpers/CacheHelper.php:140]: Its just fallback for nulls or false

Comment (Karol-Stelmaczonek) [reply helpers/CacheHelper.php]: I've checked the message and it returned int, so I changed the logic

Comment (Karol-Stelmaczonek) [reply helpers/CacheHelper.php]: I've checked the message and it returned int, so I changed the logic

**Review by pribi — APPROVED** [review]
LGTM, but not tested.

**Pull request merged**